src/inputs/button/buttonConstants.js

```diff
@@ -56,6 +61,10 @@ export const buttonPropTypes = {
      */
     style: PropTypes.shape({}),
     target: PropTypes.oneOf(['_blank']),
+    /**
+     * If `true`, the button's text is only showed.
```

Comment:
Maybe: If \`true\`, only the button's text is shown.

Reply:
Thanks!

Reply:
Fixed.

Reply:
Lookin' good.  Thank you sir!  Resolving.